service.datadog.yaml

```diff
@@ -0,0 +1,15 @@
+---
+schema-version: v2
+dd-service: datadog-agent-nightly
+
+extensions:
+  datadoghq.com/sdp:
+    workday_team: "Agent Platform"
+    conductor:
+      slack: "datadog-agent-pipelines"
+      schedule: "0 3 3 * * 1-5"
```

Comment:
Is it supposed to be a "regular" cron schedule expression? If yes I guess there are too many fields
```
minute/hour/day(month)/month/day(week)
```

Reply:
nice catch!